docs/source/migration.rst

```diff
@@ -0,0 +1,503 @@
++-----------------------------------------------+--------------------------------------+
+| ``update_runs``                               | N/A                                  |
++-----------------------------------------------+--------------------------------------+
+| ``flow_action_log``                           | ``FlowsClient.get_run_logs``         |
++-----------------------------------------------+--------------------------------------+
+| ``scope_for_flow``                            | ``SpecificFlowClient.scopes`` [*]_   |
++-----------------------------------------------+--------------------------------------+
+| ``flow_action_resume``                        | ``SpecificFlowClient.resume_run``    |
++-----------------------------------------------+--------------------------------------+
+| ``run_flow``                                  | ``SpecificFlowClient.run_flow``      |
++-----------------------------------------------+--------------------------------------+
+
+
+Converting YAML to JSON
+-----------------------
+
+In the Globus Automate Client, users were allowed to use YAML files to define
+their flows.
+However, the Flows service only accepts JSON data, and YAML was being converted
+to JSON by the client.
+
+Unfortunately, the YAML language specification contains ambiguities, and
+different parsers may treat identical documents differently.
+``globus-cli`` and ``globus-sdk`` do not support YAML parsing, but it is possible
+to convert YAML to JSON using a variety of tools.
+This approach ensures that Globus provided software operates consistently, and
+allows users to continue using YAML or to move off of it, as they prefer.
+
+In this section, we will cover two popular tools for converting YAML to JSON,
+yq (written in Go) and remarshal (written in Python). We will also cover
+one python library, ``pyyaml``, which can be used to load YAML data and pass it
+to the ``globus-sdk``.
+Various other tools provide similar functionality in other languages, and there
+are alternative parsers available in python.
+
+yq
+~~
+
+The `yq <https://mikefarah.gitbook.io/yq/>`_ tool is a CLI utility similar to the
+popular ``jq`` command.
+It provides a wide variety of commands for manipulating and extracting data
+from YAML documents.
+
+Installation
+++++++++++++
+
+``yq`` is distributed for multiple platforms as a binary as well as Homebrew
+and Snapcraft.
+For these latter two, use:
+
+.. code-block:: bash
+
+    # macOS, brew
+    brew install yq
+
+    # ubuntu-based linux, snap
+    snap install yq
+
+To get a ``yq`` binary directly, or to use the docker based distribution,
+follow
+`yq's installation instructions <https://github.com/mikefarah/yq/#install>`_.
+
+Usage
++++++
+
+In order to convert a flow from YAML to JSON using ``yq``, all that is needed
+is a command which loads the YAML document and then outputs it as JSON.
+
+.. code-block:: bash
+
+    yq -o=json foo.yaml > foo.json
```

Comment:
`bash` works here, but it might be more appropriate to highlight this (and, I think, most of the examples here) as `console`.

https://pygments.org/docs/terminal-sessions/

```suggestion
.. code-block:: console

    $ yq -o=json foo.yaml > foo.json
```

Reply:
Good call! `console` is much better than `bash` for this.